javascript/gdx_getting_started_1.html

```diff
@@ -0,0 +1,80 @@
+  <!--
+  This example uses the GDX JavaScript library to collect all of the sensor information from a Vernier Go Go Direct® Sensor.
+  Press the 'Select Go Direct Device' button to open communication to the device in a Chrome browser. 
+  This example does not collect data from a sensor. 
+  
+  This program uses the following functions from the GDX Library:
+  
+  gdxDevice.selectDevice();
+  gdxDevice.close();
+  gdxDevice.sensors.filter();
+  -->  
+
+
+<!doctype html>
+<html>
+<head>
+  <title>Example 1: Showing Sensor Information</title>
+  <meta name="description" content="The gdx_getting_started_1.html example shows how you can use the gdx functions to list information on all of the sensor channels on a Go Direct device. This includes sensor channel number, sensor description, units, and 'incompatible sensors'">
+  <meta charset="utf-8">
+  <script src="https://unpkg.com/@vernier/godirect/dist/godirect.min.umd.js"></script>
+</head>
+
+<body>
+  <button id="select_device">Select Go Direct Device</button>
+  <pre id="output"></pre>
+
+  <script>
+
+    const selectDeviceBtn = document.querySelector('#select_device');
+    const output = document.querySelector('#output');
+    const selectDevice = async () => {
+      try {
+        const gdxDevice = await godirect.selectDevice();
+
+        // print name, serial number, order code and measurement period
+        output.textContent += `\n Connected to `+gdxDevice.name;
+        output.textContent += `\n Serial Number: `+gdxDevice.serialNumber;
+        output.textContent += `\n Order Code: `+gdxDevice.orderCode;
+        output.textContent += `\n measurementPeriod: `+gdxDevice.measurementPeriod ;
+      
+       
+        // wait for battery level and battery state and print
```

Comment:
I get a timeout error here
 measurementPeriod: 500Error: write command timed out after 5s. Command: 10 Rolling Counter: f1
I think I saw code that corrected this from Jenny and Laura...but it is not here.

Reply:
sensor timeout on windows, likely a library issue.